keyboards/numatreus/keymaps/hdbx/keymap.c

```diff
@@ -0,0 +1,237 @@
+// WindowsでJIS配列として認識しているときに、オシャレなUS配列用キーキャップを使うためのキーマップです。
+// Google日本語入力の場合、以下のキー設定を行うとLower,Raiseのタップでそれぞれ半角入力と全角入力を切り替えできるようになります。
+//  ・Henkan（入力文字なし/直接入力）に「IMEを有効化」を割り当て
+//  ・Muhenkan（入力文字なし/直接入力）に「IMEを無効化」を割り当て
+
+#include QMK_KEYBOARD_H
+#include "keymap_jp.h"       // qmk_firmware-master/quantum/keymap_extras/keymap_jp.h 日本語キーボード設定用
+#include <sendstring_jis.h>  // macro sendstring for jis keyboard マクロ文字列送信時に日本語キーボード設定での文字化け回避
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layer_number {
+    _QWERTY = 0,
+    _HDBX,
+    _LOWER,
+    _RAISE,
+    _GAME,
+    _ADJUST
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  HDBX,
+  MCR1,                // マクロ1
+  MCR2,                // マクロ2
+  MCR3,                // マクロ3
+  MCR4,                // マクロ4
+  MCR5,                // マクロ5
+  DYNAMIC_MACRO_RANGE, // ダイナミックマクロ
+  WN_SCLN              // タップでJISの「:」  シフトでJISの「;」 (Windows)
+};
+
+// Use Dynamic macro
+#include "dynamic_macro.h"
```

Comment:
We converted dynamic macros to a core feature, so this is no longer valid.

```suggestion
```

Instead, you can add `DYNAMIC_MACRO_ENABLE = yes` to your rules.mk, and it will add handle all of the stuff needed for this feature.